.github/CODEOWNERS

```diff
@@ -1 +1,2 @@
 * @confluentinc/clients
+* @confluentinc/data-governance
```

Comment:
I think it should be `* @confluentinc/clients @confluentinc/data-governance` since the last one takes precedence, so we need to include both at once.